tests/functional/modules/test_zos_mvs_raw_func.py

```diff
@@ -123,57 +138,63 @@ def test_dispositions_for_existing_data_set(ansible_zos_module, disposition):
-                    "disposition":"new",
-                    "return_content":{
-                        "type":"text"
+    try:
+        hosts = ansible_zos_module
+        tmphlq = "TMPHLQ"
+        volumes = Volume_Handler(volumes_on_systems)
+        default_volume = volumes.get_available_vol()
+        default_data_set = get_tmp_ds_name()[:25]
+        hosts.all.zos_data_set(
+            name=default_data_set, type="seq", state="present", replace=True
+        )
+        idcams_dataset, idcams_args = get_temp_idcams_dataset(hosts)
+
+        results = hosts.all.zos_mvs_raw(
+            program_name="idcams",
+            auth=True,
+            tmp_hlq=tmphlq,
+            dds=[
+                {
+                    "dd_data_set":{
+                        "dd_name":SYSPRINT_DD,
+                        "data_set_name":default_data_set,
+                        "disposition":"new",
+                        "return_content":{
+                            "type":"text"
+                        },
+                        "replace":True,
+                        "backup":True,
+                        "type":"seq",
+                        "space_primary":5,
+                        "space_secondary":1,
+                        "space_type":"m",
+                        "volumes":default_volume,
+                        "record_format":"fb"
                     },
-                    "replace":True,
-                    "backup":True,
-                    "type":"seq",
-                    "space_primary":5,
-                    "space_secondary":1,
-                    "space_type":"m",
-                    "volumes":default_volume,
-                    "record_format":"fb"
-                },
-            },
-            {
-                "dd_input":{
-                    "dd_name":SYSIN_DD,
-                    "content":IDCAMS_STDIN
-                }
-            },
-        ],
-    )
-    for result in results.contacted.values():
-        assert result.get("ret_code", {}).get("code", -1) == 0
-        assert len(result.get("dd_names", [])) > 0
-        for backup in result.get("backups"):
-            backup.get("backup_name")[:6] == tmphlq
-    results = hosts.all.zos_data_set(name=default_data_set, state="absent")
-    for result in results.contacted.values():
-        assert result.get("changed", False) is True
+                },
+                {
+                    "dd_input":{
+                        "dd_name":SYSIN_DD,
+                        "content":idcams_args
+                    }
+                },
+            ],
+        )
+        for result in results.contacted.values():
+            assert result.get("ret_code", {}).get("code", -1) == 0
+            assert len(result.get("dd_names", [])) > 0
+            for backup in result.get("backups"):
+                backup.get("backup_name")[:6] == tmphlq
+        results = hosts.all.zos_data_set(name=default_data_set, state="absent")
```

Comment:
a nit-pick here, i know you didnt change this, but for the sake of consistency, can this line deleting `default_data_set` be moved into the `finally` block?